src/pages/EntityDetails/_entity-details.scss

```diff
@@ -212,79 +212,84 @@
   }
 
   .entity-details__apps {
-    th,
-    td {
-      &:nth-child(1) {
-        width: 20%;
-      }
-
-      &:nth-child(2) {
-        width: 15%;
-      }
-
-      &:nth-child(3) {
-        width: 15%;
-      }
-
-      &:nth-child(4) {
-        width: 10%;
-      }
+    &:not(.selectable) {
+      th,
+      td {
+        // Version.
+        &:nth-child(3) {
+          width: 10%;
+        }
 
-      &:nth-child(5) {
-        width: 15%;
-      }
+        // Scale.
+        &:nth-child(4) {
+          width: 10%;
+        }
 
-      &:nth-child(6) {
-        width: 10%;
-      }
+        @include mobile-and-medium {
+          // Scale.
+          &:nth-child(4) {
+            width: 15%;
+          }
 
-      &:nth-child(7) {
-        width: 10%;
-      }
+          // Version.
+          &:nth-child(3),
+          // Revision.
+          &:nth-child(5),
+          // Store.
```

Comment:
I think the comments should be the other way around, as Store is 6th and Revision is 7th.